config.yml

```diff
@@ -4560,8 +4905,8 @@ production_recipes:
     production_time: 32
     inputs:
       Clay:
-        material: CLAY
-        amount: 128
+        material: CLAY_BALL
+        amount: 512

```

Comment:
Pretty sure this is wrong. We moved to clay blocks for these recipes intentionally